### PR TITLE
feat(react-native): update @react-native-community/cli to 10.2.1

### PR DESCRIPTION
**Pull request by xiongemi**
## Related Issue(s)

Fixes #

### CODE REVIEW

**Review by jaysoo — APPROVED** [review]

File: docs/generated/packages/react-native/documents/overview.md [hunk 1]
```diff
@@ -52,36 +52,94 @@ yarn add --dev @nrwl/react-native
...
+
+To build your app and starts it on a connected Android emulator or device:
+
+```shell
+nx run-android <your-app-name>
+```
+
+### Build iOS
+
+To builds IOS app:
```
Comment (jaysoo) [inline docs/generated/packages/react-native/documents/overview.md:120]: typo: `iOS` not `IOS`.

Comment (xiongemi) [reply docs/generated/packages/react-native/documents/overview.md]: fixed.

**Review by ndcunningham — APPROVED** [review]

File: docs/shared/packages/react-native/react-native-plugin.md [hunk 2]
```diff
@@ -52,36 +52,94 @@ yarn add --dev @nrwl/react-native
...
+
+#### Use upgrade-native Generator
+
+To upgrade native iOS and Android code to latest, you can use the [upgrade-native](/packages/react-native/generators/upgrade-native) generator:
+
+```shell
+nx generate @nrwl/react-native:upgrade-native <your-app-name>
+```
+
+This is a command that will replace the iOS and Android entirely.
```
Comment (isaacplmann) [inline docs/shared/packages/react-native/react-native-plugin.md:88]: Replace the iOS and Android what?

File: docs/shared/packages/react-native/react-native-plugin.md [hunk 3]
```diff
@@ -52,36 +52,94 @@ yarn add --dev @nrwl/react-native
...
+
+#### Upgrade Manually
+
+You can also upgrade React Native iOS and Android code use project [rn-diff-purge](https://react-native-community.github.io/upgrade-helper/).
```
Comment (isaacplmann) [inline docs/shared/packages/react-native/react-native-plugin.md:92]:
```suggestion
You can also upgrade React Native iOS and Android code using the [rn-diff-purge](https://react-native-community.github.io/upgrade-helper/) project.
```

File: docs/shared/packages/react-native/react-native-plugin.md [hunk 4]
```diff
@@ -52,36 +52,94 @@ yarn add --dev @nrwl/react-native
...
+
+To start the server that communicates with connected devices:
+
+```shell
+nx start <your-app-name>
+```
+
+### Run iOS
+
+To build your app and starts it on iOS simulator or device:
```
Comment (isaacplmann) [inline docs/shared/packages/react-native/react-native-plugin.md:104]:
```suggestion
To build your app and start it on iOS simulator or device:
```

File: docs/shared/packages/react-native/react-native-plugin.md [hunk 5]
```diff
@@ -52,36 +52,94 @@ yarn add --dev @nrwl/react-native
...
+
+```shell
+nx run-ios <your-app-name>
+```
+
+### Run Android
+
+To build your app and starts it on a connected Android emulator or device:
```
Comment (isaacplmann) [inline docs/shared/packages/react-native/react-native-plugin.md:112]:
```suggestion
To build your app and start it on a connected Android emulator or device:
```

File: docs/shared/packages/react-native/react-native-plugin.md [hunk 6]
```diff
@@ -52,36 +52,94 @@ yarn add --dev @nrwl/react-native
...
+
+```shell
+nx run-android <your-app-name>
+```
+
+### Build iOS
+
+To builds iOS app:
```
Comment (isaacplmann) [inline docs/shared/packages/react-native/react-native-plugin.md:120]:
```suggestion
To build an iOS app:
```

File: docs/shared/packages/react-native/react-native-plugin.md [hunk 7]
```diff
@@ -52,36 +52,94 @@ yarn add --dev @nrwl/react-native
...
+
+```shell
+nx build-ios <your-app-name>
+```
+
+The build artifacts will be located under `<your app folder>/ios/build`.
+
+You can specify the build folder by passing option `buildFolder`:
```
Comment (isaacplmann) [inline docs/shared/packages/react-native/react-native-plugin.md:128]:
```suggestion
You can specify the build folder by setting the `buildFolder` option:
```

File: docs/shared/packages/react-native/react-native-plugin.md [hunk 8]
```diff
@@ -52,36 +52,94 @@ yarn add --dev @nrwl/react-native
...
+
+```shell
+nx build ios <your-app-name> --buildFolder="./build"
+```
+
+### Build Android
+
+To build Android app, run:
```
Comment (isaacplmann) [inline docs/shared/packages/react-native/react-native-plugin.md:136]:
```suggestion
To build an Android app, run:
```

**Review by isaacplmann — CHANGES_REQUESTED** [review]

File: docs/shared/packages/react-native/react-native-plugin.md (hunk 3 above)
Comment (xiongemi) [reply docs/shared/packages/react-native/react-native-plugin.md:92]: done.

File: docs/shared/packages/react-native/react-native-plugin.md (hunk 4 above)
Comment (xiongemi) [reply docs/shared/packages/react-native/react-native-plugin.md:104]: changed

File: docs/shared/packages/react-native/react-native-plugin.md (hunk 5 above)
Comment (xiongemi) [reply docs/shared/packages/react-native/react-native-plugin.md:112]: changed

File: docs/shared/packages/react-native/react-native-plugin.md (hunk 6 above)
Comment (xiongemi) [reply docs/shared/packages/react-native/react-native-plugin.md]: changed.

File: docs/shared/packages/react-native/react-native-plugin.md (hunk 7 above)
Comment (xiongemi) [reply docs/shared/packages/react-native/react-native-plugin.md]: changed.

**Review by isaacplmann — APPROVED** [review]

**Pull request merged**